### PR TITLE
Restore sanitised db using flask cli

### PR DESCRIPTION
**Pull request by israr-ulhaq**
I created a Flask CLI command that allows you to restore a sanitised database directly from the production bucket to your local database. I also added a help function and included named arguments for better usability.

### Change description
_A brief description of the pull request_

- [ ] Unit tests and other appropriate tests added or updated
- [x] README and other documentation has been updated / added (if needed)
- [ ] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
_If manual testing is needed, give suggested testing steps_

### CODE REVIEW

**Review by wjrm500 — CHANGES_REQUESTED** [review]
Hi again @israr-ulhaq, thanks for renaming that variable. I've left one comment around using a named argument for the `restore-sanitised-database.sh` script, which I feel is important for clarity. Also we should make sure to squash all commits together on merge as they do not make sense in isolation in their current order - [adding the CLI command](https://github.com/communitiesuk/funding-service-design-post-award-data-store/pull/794/commits/726c61baf8d8fa291b25a734cff28ca5d0fa9c0e) that uses the new `y` arg on the `restore-sanitised-database.sh` script only works if that [script has been updated to support it](https://github.com/communitiesuk/funding-service-design-post-award-data-store/pull/794/commits/7c433f122ce541dbc63a597b3e708dc47b10e0c6)

File: scripts/restore-sanitised-database.sh [hunk 1]
```diff
@@ -6,6 +6,7 @@ LOCAL_DB_PASSWORD="password"
 LOCAL_DB_USER="postgres"
 LOCAL_DB_NAME="data_store"
 DB_IMAGE="postgres:16.2"
+RESTORE_TO_LOCAL_DB=${1}
```
[the remark this answers is not in the document]
Comment (israr-ulhaq) [reply scripts/restore-sanitised-database.sh]: Done!

**Review by wjrm500 — APPROVED** [review]
Hey @israr-ulhaq, have tested locally and works well! I like that you now need to explicitly specify that you want to restore to the cloud, that feels safer

I do have a big question over the general process here, which is: How can you get a sanitised prod snapshot onto the dev or test AWS DB, or into your local DB, when the `sanitise-post-award-database.sh` script saves the prod snapshot to prod S3, and the `restore-sanitised-database.sh` script doesn't allow prod access?

But this isn't strictly relevant to this PR so approving regardless. Please rebase main before merge

**Comment by israr-ulhaq** [comment]
> Hey @israr-ulhaq, have tested locally and works well! I like that you now need to explicitly specify that you want to restore to the cloud, that feels safer
> 
> I do have a big question over the general process here, which is: How can you get a sanitised prod snapshot onto the dev or test AWS DB, or into your local DB, when the `sanitise-post-award-database.sh` script saves the prod snapshot to prod S3, and the `restore-sanitised-database.sh` script doesn't allow prod access?
> 
> But this isn't strictly relevant to this PR so approving regardless. Please rebase main before merge

Hi @wjrm500, thanks for reviewing the PR! Great question about how we’re fetching the file from the prod S3 bucket, even though this script doesn’t have prod access directly to restore db.

So the `sanitise-post-award-database.sh` script sanitises the prod snapshot and uploads the `Sanitised-database.sql` file to S3 bucket in prod. To access this file from a non-prod env, @gidsg set up a this cross-account policy [here](https://github.com/communitiesuk/funding-service-design-workflows/blob/main/apps/pre-award/copilot/environments/addons/sanitised-database-dumps.yml#L45-L98) , which allows dev&test to access this specific bucket where this `Sanitised-database.sql` file is stored.

**Pull request merged**